samples/GalleryApp/MainPage.xaml

```diff
@@ -9,57 +9,67 @@
-                    <TextBlock Text="A set of sample apps that can detect and identify the poses of the bodies of individuals in a video or image." 
-                               Style="{StaticResource BodyTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 60, 0, 0"
-                               TextWrapping="Wrap"/>
-                </Grid>
-                <Button Height="400" Width="400" Opacity="0.5" Click="NavigateToSkeletalDetectorPage" Style="{StaticResource ButtonRevealStyle}"/>
-            </Grid>
-
-            <Grid Grid.Column="2" Background="LightGray" Margin="25" Height="400" 
-                  Width="400" HorizontalAlignment="Left" VerticalAlignment="Top">
-                <Grid Height="150" Width="400" VerticalAlignment="Bottom">
-                    <Rectangle Height="200" Width="400" VerticalAlignment="Bottom"
-                                                       Fill="Gray" Opacity="0.75"/>
-                    <TextBlock Text="Object Detector" 
-                               Style="{StaticResource SubheaderTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 0, 0, 0" />
-                    <TextBlock Text="A set of sample apps that detect and classify 80 objects using the Object Detector NuGet package." 
-                               Style="{StaticResource BodyTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 60, 0, 0"
-                               TextWrapping="Wrap"/>
-                </Grid>
-                <Button Height="400" Width="400" Opacity="0.5" Click="NavigateToObjectDetectorPage" Style="{StaticResource ButtonRevealStyle}"/>
-            </Grid>
+            <!--Category bar-->
+            <controls:TabView x:Name="Tabs"
+                Grid.Column="1" 
+                TabWidthBehavior="Actual"
+                CanCloseTabs="False"
+                IsCloseButtonOverlay="False"
+                CanDragItems="False"
+                CanReorderItems="False"
+                AllowDrop="False">
+                
+                <!-- Tabs -->
+                <controls:TabViewItem Header="Vision" Icon="Camera">
+                    <GridView x:Name="ImageGridView" ItemsSource="{x:Bind SkillPages}"
+                        Margin="20" IsItemClickEnabled="True"                     
+                        ItemTemplate="{StaticResource ImageGridView_DefaultItemTemplate}"
+                        ItemContainerStyle="{StaticResource ImageGridView_DefaultItemContainerStyle}"
+                        ItemClick="SampleGridView_ItemClick">
+                    </GridView>
 
+                </controls:TabViewItem>
+                <controls:TabViewItem Header="Audio" Icon="Audio">
+                    <TextBlock Padding="16">Coming soon...</TextBlock>
+                </controls:TabViewItem>
```

Comment:
A separate task to load tab view using template & data binding.
Task 22514227: Load category tabs in MainPage using template & data binding